ui/src/app/machines/components/ActionFormWrapper/ReleaseForm/ReleaseForm.tsx

```diff
@@ -0,0 +1,111 @@
+import { useEffect } from "react";
+
+import { Spinner, Strip } from "@canonical/react-components";
+import PropTypes from "prop-types";
+import { useDispatch, useSelector } from "react-redux";
+import * as Yup from "yup";
+
+import ReleaseFormFields from "./ReleaseFormFields";
+
+import ActionForm from "app/base/components/ActionForm";
+import { useMachineActionForm } from "app/machines/hooks";
+import { config as configActions } from "app/settings/actions";
+import configSelectors from "app/store/config/selectors";
+import type { MachineAction } from "app/store/general/types";
+import { actions as machineActions } from "app/store/machine";
+import machineSelectors from "app/store/machine/selectors";
+import { NodeActions } from "app/store/types/node";
+
+export type ReleaseFormValues = {
+  enableErase: boolean;
+  quickErase: boolean;
+  secureErase: boolean;
+};
+
+const ReleaseSchema = Yup.object().shape({
+  enableErase: Yup.boolean(),
+  quickErase: Yup.boolean(),
+  secureErase: Yup.boolean(),
+});
+
+type Props = {
+  setSelectedAction: (action: MachineAction | null, deselect?: boolean) => void;
```

Comment:
Is this different to the `SetSelectedAction` type in `ui/src/app/machines/views/MachineDetails/types.ts`? If not that could be used here.

Reply:
Nope, it's the same. Thanks!